src/Honeybee_ Run Energy Simulation.py

```diff
@@ -917,15 +948,17 @@ def writeBatchFile(self, workingDir, idfFileName, epwFileAddress, EPDirectory =
         folderName = workingDir.replace( (workingDrive + '\\'), '')
         batchStr = workingDrive + '\ncd\\' +  folderName + '\n' + EPDirectory + \
                 '\Epl-run ' + fullPath + ' ' + fullPath + ' idf "' + epwFileAddress + '" EP N nolimit N N 0 Y'
-    
+        
+        #print "this is batch string" + batchStr
         batchFileAddress = fullPath +'.bat'
         batchfile = open(batchFileAddress, 'w')
         batchfile.write(batchStr)
         batchfile.close()
         
         #execute the batch file
-        os.system(batchFileAddress)
-        #os.system('C:\\honeybee\\runIt.bat')
+        os.system(batchFileAddress) 
+
+        os.system('C:\\honeybee\\runIt.bat')
```

Comment:
@antonszilasi you need to use Honeybee's and project's default folder and not c:\honeybee. This is a bug.